jose.py

```diff
@@ -35,13 +35,15 @@
     'br?',
     'u died [real] [Not ClickBait]',
     'rEEEEEEEEEEEEE',
+    'you didn\'t buy gold',
```

Comment:
This is a mostly obscure, mwk meme, and people outside mwk will just go "owo wats this", so IDK if this makes sense to have.